roles/siege-run/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+# Coppied from mycourses load-generator.yml example
+- name: Configure Siege
+  template:
+    src: "files/siege.conf"
+    dest: "/.sigerc"
+    seuser: system_u
+
+- name: Push hosts file
+  template:
+    src: "files/hosts.conf"
+    dest: "/usr/local/etc/urls.txt"
+    seuser: system_u
+
+- name: Generating Load
+  command: "siege"
```

Comment:
Do we need to pass any arguments to the Siege command to work correctly, e.g. using our custom `.siegerc` file?

Reply:
Nope, should just pick the defaults from /.siegerc. Theoretically.